packages/cli-kit/src/public/node/system.ts

```diff
@@ -181,3 +181,12 @@ export function terminalSupportsPrompting(): boolean {
   }
   return Boolean(process.stdin.isTTY && process.stdout.isTTY)
 }
+
+/**
+ * Check if the current environment is a CI environment.
+ *
+ * @returns True if the current environment is a CI environment.
+ */
+export function isCI(): boolean {
```

Comment:
Surprising that we didn't have this function. Could you update the places where we directly use `process.env.CI`?